### PR TITLE
update scaleway zones

### PR DESCRIPTION
**Pull request by nikaro**
##### SUMMARY
Scaleway provides some new zones since a while: https://developers.scaleway.com/en/products/instance/api/#endpoints

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
scaleway

##### ADDITIONAL INFORMATION
None.

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Please make sure that the sanity tests pass (there are a lot of modules which also need updating).

Also, please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

**Comment by nikaro** [comment]
@felixfontein it looks good this time.

File: changelogs/fragments/1690-scaleway-regions.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway - update regions and zones to add the new ones.
```
Comment (felixfontein) [inline changelogs/fragments/1690-scaleway-regions.yaml:2]:
```suggestion
  - scaleway modules and inventory plugin - update regions and zones to add the new ones (https://github.com/ansible-collections/community.general/pull/1690).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@DenBeke @Spredzy dear maintainers, could you please take a quick look? Since we're releasing community.general 2.0.0 in a few hours, this could make it in. I'm currently leaning to merge this since it shouldn't break anything, but a more qualified look would be awesome!

**Comment by remyleone** [comment]
LGTM

**Comment by felixfontein** [comment]
@nikaro thanks for fixing this!
@remyleone thanks for reviewing!

**Pull request merged**